### DISCUSSION TITLE
Selecting backend in Colab

### DISCUSSION THREAD

**Discussion by nirmal129** [lululxvi/deepxde#1250] [Q&A] [answered]
Hello, 

Sorry for this silly question but I am confused about selecting the backend in Colab. Whenever I run DeepXDE on Colab, it notifies me that the backend is not selected and thus uses TensorFlow.1. Is there any way to change the backend on Colab? I didn't find any way to do it in the documentation or GitHub discussion. 

Thank you!

**Comment by tsarikahin** [comment, marked as answer]
https://github.com/lululxvi/deepxde/issues/947
